### PR TITLE
remove mentions of legacy mysql adapter from guides

### PR DESCRIPTION
**Pull request by bronson**

### CODE REVIEW

**Comment by maclover7** [comment]
Can you please rebase?

**Comment by bronson** [comment]
happy to, ready now.

File: guides/source/development_dependencies_install.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ $ bundle exec ruby -Itest path/to/test.rb -n test_name
 
 ### Active Record Setup
 
-The test suite of Active Record attempts to run four times: once for SQLite3, once for each of the two MySQL gems (`mysql` and `mysql2`), and once for PostgreSQL. We are going to see now how to set up the environment for them.
+Active Record's test suite runs three times: once for sqlite3, once for mysql2, and once for postgresql. We are going to see now how to set up an environment for each of them.
```
Comment (matthewd) [inline guides/source/development_dependencies_install.md]: I prefer the previous phrasing here: we use the official software names where possible, and "the environment" is more accurate.

**Comment by bronson** [comment]
Sounds good, updated.

**Pull request merged**